tests/macros/utils.js

```diff
@@ -1,86 +1,243 @@
+/// <reference path="./utils.doc.js"/>
 /**
  * @prettier
  */
 
 // Provides utilities that as a whole constitute the macro test framework.
 const { execSync } = require('child_process');
+const fs = require('fs');
 const os = require('os');
+const path = require('path');
 
 const vnu = require('vnu-jar');
 
 const Environment = require('../../src/environment.js');
 const Templates = require('../../src/templates.js');
 
-// When we were doing mocha testing, we used this.macro to hold this.
-// But Jest doesn't use the this object, so we just store the object here.
+/**
+ * When we were doing mocha testing, we used this.macro to hold this.
+ * But Jest doesn't use the this object, so we just store the object here.
+ * @type {Macro}
+ */
 let macro = null;
 
+/**
+ * Asserts that value is truthy.
+ *
+ * @template T Type of value.
+ * @param {T} x Actual value.
+ */
 function assert(x) {
-    expect(x).toBe(true);
+    expect(x).toBeTruthy();
 }
 
+/**
+ * Asserts non-strict equality (==) of actual and expected.
+ *
+ * @template T Type of the objects.
+ * @param {T} x Actual value.
+ * @param {any} y Potential expected value.
+ */
 assert.equal = (x, y) => {
     expect(x).toEqual(y);
 };
 
 assert.eventually = {
+    /**
+     * Asserts non-strict equality (==) of actual and expected.
```

Comment:
This is not true. Jest's toEqual() matcher does deep equality comparison using Object.is()